bin/sozo/src/ops/migration/mod.rs

```diff
@@ -358,11 +328,21 @@ where
             if let Some(meta) = metadata.as_ref().and_then(|inner| inner.world()) {
                 match meta.upload().await {
                     Ok(hash) => {
-                        let encoded_uri = cairo_utils::encode_uri(&format!("ipfs://{hash}"))?;
+                        let mut encoded_uri = cairo_utils::encode_uri(&format!("ipfs://{hash}"))?;
+
+                        // Metadata is expecting an array of capacity 3.
```

Comment:
this is for now hard coded, we should find a better way for sozo to know this interactively by fetching the model data.
Maybe good for an other PR.